config/deny-all.json

```diff
@@ -0,0 +1,10 @@
+{
+	"defaultAction": "SCMP_ACT_ERRNO",
+	"architectures": [
+		"SCMP_ARCH_X86_64",
+		"SCMP_ARCH_X86",
+		"SCMP_ARCH_X32"
+	],
+	"syscalls": [
+	]
+}
```

Comment:
This would allow no syscalls from containers started with this profile! Not sure even if the container would start.

Reply:
Yep this is the intention until we are able to get more specific